main.js

```diff
@@ -1174,29 +1175,119 @@ function cmd_diffsend(lang, msg, args, wiki, reaction, spoiler) {
+								console.log( '- This wiki doesn\'t exist! ' + ( cperror ? cperror.message : ( cpbody ? ( cpbody.error ? cpbody.error.info : '' ) : '' ) ) );
+								msg.reactEmoji('nowiki');
+							}
+							else {
+								console.log( '- ' + ( cpresponse ? cpresponse.statusCode + ': ' : '' ) + 'Error while getting the search results' + ( cperror ? ': ' + cperror : ( cpbody ? ( cpbody.error ? ': ' + cpbody.error.info : '.' ) : '.' ) ) );
+								msg.sendChannelError( spoiler + '<' + wiki.toLink() + 'Special:Diff/' + ( args[1] ? args[1] + '/' : '' ) + args[0] + '>' + spoiler );
+							}
+						} else {
+							var content_diff = cpbody.compare["*"];
+							var current_tag = "";
+							var small_prev_ins = "";
+							var small_prev_del = "";
+							var deleted = false;
+							var addedto = false;
+							var added = 0
+							var parser = new htmlparser.Parser({
+								onopentag: function(name, attribs){
+									console.log("onopentag name: " + name + " attribs: "+ JSON.stringify(attribs, null, 4)); // TODO Remove debug
+									if(name === "ins" || name == "del"){
+										current_tag = name;
+									}
+									if(name === "td" && attribs.class === "diff-addedline"){
+										current_tag = name+"a";
+									}
+									if(name === "td" && attribs.class === "diff-deletedline"){
+										current_tag = name+"d";
+									}
+									if(name === "td" && attribs.class === "diff-marker"){
+										added = 1
+										console.log("added=1"); // TODO Remove debug
+									}
+								},
+								ontext: function(text){
+									if (current_tag === "ins" && small_prev_ins.length < 1000) {
+										small_prev_ins = small_prev_ins + "**" + text.escapeFormatting() + "**";
+										addedto = true;
+									}
+									if (current_tag === "del" && small_prev_del.length < 1000) {
+										small_prev_del = small_prev_del + "~~" + text.escapeFormatting() + "~~";
+										addedto = true;
+									}
+									if (current_tag === "afterins" || current_tag === "tda") {
+										small_prev_ins = small_prev_ins + text.escapeFormatting();
+									}
+									if (current_tag === "afterdel" || current_tag === "tdd") {
+										small_prev_del = small_prev_del + text.escapeFormatting();
+									}
+									if (added === 1 && text === "+") {
+										added = 2;
+										console.log("added=2"); // TODO Remove debug
+									}
+									if (added === 1 && text === "-") {
+										added = 3;
+										console.log("added=3"); // TODO Remove debug
+									}
+									console.log("text: " + text) // TODO Remove debug
+								},
+								onclosetag: function(tagname){
+									console.log("onclosetag name: " + tagname); // TODO Remove debug
+									if (tagname === "ins") {
+										current_tag = "afterins";
+									} else if (tagname === "del") {
+										current_tag = "afterdel";
+									} else if (tagname === "tr" && added === 2) {
```

Comment:
This if is problematic, if the edit only removed lines without adding anything the diff in removed side won't have line breaks